src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java

```diff
@@ -807,7 +808,7 @@ public void verifyMainEditEditor(Editor expectedEditor) {
         break;
       case CK:
         VisualEditModePageObject ck = openCKModeWithMainEditButton();
-        ck.verifyContentLoaded();
+        Assertion.assertTrue(ck.isContentLoaded(), "Content is not loaded");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Content is not loaded" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)